test/src/unit-msgpack.cpp

```diff
@@ -34,68 +34,68 @@ using nlohmann::json;
 
 #include <fstream>
 
-class SaxCountdown : public nlohmann::json::json_sax_t
+class SaxCountdown
```

Comment:
Why aren't we inheriting here any more?

Reply:
I did remove inheritance from tests, but since we decided to expose a base class, I shall add a test using this base class.

I will be able to work on the PR on Thursday